packages/renderer/src/lib/preferences/PreferencesResourcesRendering.svelte

```diff
@@ -0,0 +1,187 @@
+                        <div class="text-center mt-10">
+                            <!-- create new podman machine button -->
+                            <button
+                                class="pf-c-button pf-m-primary "
+                                title="Create new Podman machine"
+                                type="button">
+                                Create new ...
+                            </button>
+                        </div>
+                    </div>                    
+                </div>
+                <!-- podman machines columns -->
+                {#each provider.containerConnections as container}
+                <div class="px-5 min-w-[230px]">
+                    <div class="float-right">
+                        <Fa icon="{faArrowUpRightFromSquare}" />
+                    </div>
+                    <div class="{container.status !== 'started' ? 'text-gray-500' : ''} text-sm">
+                        {container.name}
+                    </div>
+                    <div class="flex">
+                        {#if container.status === 'started'}
+                        <div class="my-auto w-3 h-3 bg-green-500 rounded-full" />
+                        <span class="my-auto text-green-500 ml-1 font-bold text-[9px]">RUNNING</span>
+                        {:else if container.status === 'starting'}
+                        <div class="my-auto w-3 h-3 bg-green-500 rounded-full" />
+                        <span class="my-auto text-green-500 ml-1 font-bold text-[9px]">STARTING</span>
+                        {:else if container.status === 'stopped'}
+                        <div class="my-auto w-3 h-3 bg-gray-500 rounded-full" />
+                        <span class="my-auto text-gray-500 ml-1 font-bold text-[9px]">OFF</span>
+                        {:else}
+                        <div class="my-auto w-3 h-3 bg-gray-500 rounded-full" />
+                        <span class="my-auto text-gray-500 ml-1 font-bold text-[9px]">{container.status.toUpperCase()}</span>
+                        {/if}
+                    </div>
+                    {#if container.machineInfo}
+                    <div class="flex mt-3 {container.status !== 'started' ? 'text-gray-500' : ''}">
+                        <div class="mr-4">
+                            <div class="text-[9px]">vCPUs</div>
+                            <div class="text-xs">{container.machineInfo.cpus}</div>
+                        </div>
+                        <div class="mr-4">
+                            <div class="text-[9px]">MEMORY</div>
+                            <div class="text-xs">{container.machineInfo.memory.toFixed(1)}MB</div>
+                        </div>
+                        <div class="mr-4">
+                            <div class="text-[9px]">DISK LIMIT</div>
+                            <div class="text-xs">{container.machineInfo.diskSize.toFixed(2)}GB</div>
+                        </div>
```

Comment:
it seems that you're removing all the generic aspect of the providers
we have properties that declare their type, range, etc
here you're hardcoding what is generic